sites/public/pages/eligibility/age.tsx

```diff
@@ -13,7 +13,6 @@ import {
 } from "@bloom-housing/ui-components"
 import FormsLayout from "../../layouts/forms"
 import { useForm } from "react-hook-form"
-import "./age.scss"
```

Comment:
Oh cause it stopped being imported here?

Reply:
Yeah, when it was imported here I got this error, which I think is caused by the legacy `@zeit/next-sass` import: 

`Global CSS cannot be imported from files other than your Custom <App>. Please move all global CSS imports to pages/_app.js. Read more: https://err.sh/next.js/css-global`

I so I moved the import to the main app, but that isn't the recommended way around the problem (so thanks for pointing this out!). I updated it to use scss modules instead, which is recommended.